src/govuk/components/pagination/template.njk

```diff
@@ -0,0 +1,71 @@
+{% set blockLevel = not params.items and (params.next or params.previous) %}
+
+<nav class="govuk-pagination {{- ' ' + params.classes if params.classes }} {{- ' govuk-pagination--block' if blockLevel }}" role="navigation" aria-label="{{ params.landmarkLabel | default("results") }} {%- for attribute, value in params.attributes %} {{ attribute }}="{{ value }}" {%- endfor -%}">
+  {%- if params.previous and params.previous.href -%}
+    <div class="govuk-pagination__prev">
+      <a class="govuk-link govuk-pagination__link" href="{{ params.previous.href }}" rel="prev" aria-label="{{ params.previous.visuallyHiddenText | default("Previous page") }}" {%- for attribute, value in params.previous.attributes %} {{ attribute }}="{{ value }}" {%- endfor -%}>
+        <svg class="govuk-pagination__icon govuk-pagination__icon--prev" xmlns="http://www.w3.org/2000/svg" height="13" width="15" aria-hidden="true" focusable="false" viewBox="0 0 15 13">
+          <path d="m6.5938-0.0078125-6.7266 6.7266 6.7441 6.4062 1.377-1.449-4.1856-3.9768h12.896v-2h-12.984l4.2931-4.293-1.414-1.414z"></path>
+        </svg>
+        <span class="govuk-pagination__link-title {{- ' govuk-pagination__link-title--decorated' if blockLevel and not params.previous.labelText }}">
+          {%- if params.previous.text -%}
+            {{- params.previous.text -}}
+          {%- else -%}
+            Previous
+          {%- endif -%}
+        </span>
+
+        {%- if params.previous.labelText and blockLevel -%}
+          <span class="govuk-visually-hidden">:</span>
+          <span class="govuk-pagination__link-label">{{ params.previous.labelText }}</span>
+        {%- endif -%}
+      </a>
+    </div>
+  {% endif %}
+
+  {%- if params.items -%}
+    <ul class="govuk-pagination__list">
+      {%- for item in params.items -%}
+        {%- if item.ellipsis -%}
+          <li class="govuk-pagination__item govuk-pagination__item--ellipses">&ctdot;</li>
+        {%- elseif item.number -%}
+          <li class="govuk-pagination__item {{- ' govuk-pagination__item--current' if item.current }}">
+            <a class="govuk-link govuk-pagination__link" href="{{ '' if item.current else item.href }}" aria-label="{{ item.visuallyHiddenText | default("Page " + item.number) }}" {%- if item.current %} aria-current="page" {% endif -%} {%- for attribute, value in item.attributes %} {{ attribute }}="{{ value }}" {%- endfor -%}>
```

Comment:
Have we decided on where the current link should go to? 

Generally I think if an `href` is set we should probably always use if rather than hard-coding the empty string for the current item.

Reply:
We discussed this on slack and we've agreed to set `items.href` on all items including the current item to required and have no logic on this `href` as all items are now links and it is the logical option to ask teams to specify the link for each item, current or not.

Reply:
Discussed this on slack. I've made the `href` option for all `item`'s required and we will be asking teams to pass `href` to all items, `current` or otherwise. We're doing this because now that the current item is a link, it makes sense for teams to treat it like a link and mange the link to the page they are currently on as part of the implementation.